hmvec/hmvec.py

```diff
@@ -452,6 +563,11 @@ def _get_term(iname):
                 rterm1 = self._get_matter(iname)
                 rterm01 = self._get_matter(iname,lowklim=True)
                 b = 1
+            elif iname in self.pk_profiles.keys():
+                rterm1 = self._get_pressure(iname)
+                rterm01 = self._get_pressure(iname,lowklim=True)
+                print ('Check the consistency relation for tSZ')
+                b = rterm01 =0
```

Comment:
I don't understand why this should be 0. What should the low-k limit of the yy power spectrum be?